parm/archive/gefs_atmos.yaml.j2

```
@@ -0,0 +1,58 @@
+{% set cycle_HH = current_cycle | strftime("%H") %}
+{% set cycle_YMD = current_cycle | to_YMD %}
+{% set cycle_YMDH = current_cycle | to_YMDH %}
+{% set head = RUN + ".t" + cycle_HH + "z." %}
+
+gefs_atmos:
+    name: "GEFS_ATMOS"
+    target: "{{ ATARDIR }}/{{ cycle_YMDH }}/gefs_atmos.tar"
+    required:
+#select mem%03d and ensstat files required
+{% set membr = ["ensstat"] %}
```

`membr` sounds like just one. We are long past the days where we were limited by the number of characters we could use in variable names; be expressive 😉.
```suggestion
{% set members = ["ensstat"] %}
```